wp-content/themes/pub/wporg-learn-2020/template-parts/content-single.php

```diff
@@ -13,10 +13,21 @@
 
 	<section>
 		<header class="row align-middle between section-heading section-heading--with-space">
-			<h1 class="section-heading_title h2"><?php the_title(); ?></h1>
+			<h1 class="section-heading_title bigger"><?php the_title(); ?></h1>
 		</header>
 
-		<div class="lp-content">
+		<hr>
+
+		<?php
+		$embed = get_media_embedded_in_content( apply_filters( 'the_content', get_the_content() ));
```

Comment:
Could you please add some testing instructions for this? A link to an existing lesson plan that has this content would help.

Reply:
@adamwoodnz You can add one WordPress.tv block for it: https://a.cl.ly/DOurWjy1

https://www.figma.com/file/UtmG8KQv3Csy8hNtuAWkwZ/Learn-WordPress?node-id=1094%3A1344 (https://a.cl.ly/2Nu7k2Dv)

Reply:
Ok thanks, I can see this working with a video embed. The Figma suggests to me that it's supposed to be a slide deck, but maybe any embed is valuable.